### PR TITLE
Translate to russian.

### PR DESCRIPTION
**Pull request by IlyaPavlovskii**
Translate to Russian [#462](https://github.com/android/sunflower/issues/462)

### CODE REVIEW

**Comment by IlyaPavlovskii** [comment]
@tiembo can you approve my pull request?

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <!-- String plurals and their related prefix / suffix strings -->
+    <string name="watering_needs_prefix">Необходим полив</string>
+    <plurals name="watering_needs_suffix">
+        <item quantity="one">раз в %d день</item>
+        <item quantity="few">каждый %d дня</item>
+        <item quantity="many">каждые %d дней</item>
+        <item quantity="other">каждые %d дней</item>
+    </plurals>
+
+    <plurals name="watering_next">
```
Comment (tiembo) [inline app/src/main/res/values-ru/strings.xml:41]: Can you remove the trailing period in lines 42 and 45?

**Comment by tiembo** [comment]
Hi @IlyaPavlovskii , apologies for the delay.  Thank you for the translation - added a comment on trailing periods.

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (IlyaPavlovskii) [reply app/src/main/res/values-ru/strings.xml:41]: Thanks. Done.

**Comment by tiembo** [comment]
Thank you! Added this to the [string translation wiki page](https://github.com/android/sunflower/wiki/Notable-Community-Contributions).

**Pull request merged**